buildenv/jenkins/JenkinsfileBase

```diff
@@ -26,7 +72,33 @@ def test() {
 					CUSTOMIZED_SDK_URL = ''
 				}
 				env.SPEC = "${SPEC}"
-				env.BUILD_LIST= "${TESTPROJECT}"
+				//check if customized BUILD_LIST is defiend
+				try{
```

Comment:
Nitpick, but please use consistent spacing ( a single space ) between keywords and braces { and brackets (

Reply:
updated